pkg/virt-handler/vm.go

```diff
@@ -3789,3 +3782,14 @@ func (d *VirtualMachineController) updateMemoryInfo(vmi *v1.VirtualMachineInstan
 	vmi.Status.Memory.GuestCurrent = currentGuest
 	return nil
 }
+
+func configureParallelMigrationThreads(options *cmdclient.MigrationOptions, vm *v1.VirtualMachineInstance) {
+	// When the CPU is limited, there's a risk of the migration threads choking the CPU resources on the compute container.
+	// For this reason, we will avoid configuring migration threads in such scenarios.
+	if cpuLimit, cpuLimitExists := vm.Spec.Domain.Resources.Limits[k8sv1.ResourceCPU]; cpuLimitExists && !cpuLimit.IsZero() {
+		return
+	}
+
+	// This value was determined after consulting with libvirt developers and performing extensive testing.
```

Comment:
Nit: it would make more sense to me to have the comment on the const rather than where it's used.
Ideally the const would also be declared in a more central location, like virt-config.

Reply:
> Nit: it would make more sense to me to have the comment on the const rather than where it's used.

Yup, that makes total sense.
Done, thank you!

> Ideally the const would also be declared in a more central location, like virt-config.

I'm not sure I agree here since virt-config mostly consists of default values while this value is fixed and static. I see it as more of an implementation detail than a user-facing API.